### PR TITLE
Added Site Name to shop pages

### PR DESCRIPTION
**Pull request by lindaicing**
For shops without a "name" property (no data yet), the siteurl changes to "Unavailable"
Screen Shot 2020-07-17 at 11 48 18 AM

On a site page with name data:
Screen Shot 2020-07-17 at 11 48 44 AM

The description doesn't show when it's not a shop:
Screen Shot 2020-07-17 at 11 49 56 AM

### CODE REVIEW

File: Extension/popup.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
     </div>
 
     <div id="extension">
+        <p id="sitename">Site Name: <span id="siteurl">Goes Here</span></p>
```
Comment (CubeOfCheese) [inline Extension/popup.html:27]: Remove the "Goes Here" or replace with "Unavailable" as a fallback

File: Extension/popupNoRating.html [hunk 2]
```diff
@@ -25,6 +25,7 @@
     </div>
 
     <div id="extension">
+        <p id="sitename">Site Name: <span id="siteurl">Goes Here</span></p>
```
Comment (CubeOfCheese) [inline Extension/popupNoRating.html:28]: Replace

**Comment by CubeOfCheese** [comment]
I think the font could be tiny bit larger if possible

**Comment by lindaicing** [comment]
Updated
Screen Shot 2020-07-17 at 9 17 04 PM

**Pull request merged**